config/initializers/extends/permission_extends.rb

```diff
@@ -44,6 +44,15 @@ module FormPermissions
     ).each { |permission| const_set(permission, "form_#{permission.parameterize}") }
   end
 
+  module FormResponsePermissions
```

Comment:
This permission group has not been added to the OWNER_PERMISSIONS below. 

It is missing migration for old already existing user roles in the database.